test/Microsoft.VisualStudio.SDK.Analyzers.Tests/VSSDK001DeriveFromAsyncPackageCodeFixTests.cs

```diff
@@ -227,13 +227,13 @@ protected override void Initialize()
     }
 }
 ";
-        var withFix = @"using System.Threading;
-using Microsoft.VisualStudio.Shell;
-using Task = System.Threading.Tasks.Task;
-
+        var withFix = @"
 namespace NS
 {
     using System;
+    using System.Threading;
```

Comment:
This change looks correct, but I'm left wondering how the test ever passed. Did the codefix actually add the `using` directives outside the namespace even though others were *inside*? Thanks for fixing this.

Reply:
Unfortunately this was unmade when reverting the other changes. My understanding is that I didn't fix it, the fix came with using the newer Roslyn packages. This is supported by the fact that we don't actively choose where to add these using, the `Microsoft.CodeAnalysis.Simplification.Simplifier` adds them automatically (except for the `Task=` one).
I won't try to fix this in this PR since it is out of scope.